### DISCUSSION TITLE
suggestion: emotion icon

### DISCUSSION THREAD

**Discussion by SilvanoGMatta** [primefaces/community#117] [PrimeIcons] [answered]
Sugest Icon Emotion
![image](https://github.com/primefaces/community/assets/25065978/97a20769-c11f-404b-928f-c3093aab9d0a)

**Comment by onursenture** [comment]
Hi, please submit your icon requests by creating issues on the PrimeIcons repository: https://github.com/primefaces/primeicons/issues

**Reply by jepsar** [reply, marked as answer]
Already requested https://github.com/primefaces/primeicons/issues/383
